test/unit/variable_definition_test.rb

```diff
@@ -0,0 +1,59 @@
+require 'test_helper'
+
+module Shipit
+  class VariableDefinitionTest < ActiveSupport::TestCase
```

Comment:
This class was completely untested. Got it passing first, then added my changes.